tests/ttnn/unit_tests/operations/backward/test_backward_addalpha.py

```diff
@@ -62,7 +62,7 @@ def test_bw_addalpha_wo_alpha(input_shapes, device):
     ),
 )
 @pytest.mark.parametrize("alpha", [0.05, 2.0, 1.5, 0.12])
-@pytest.mark.parametrize("are_required_outputs", [[True, True], [True, False], [False, True], [False, False]])
```

Comment:
Why are we removing this test case?

Reply:
Moreh team had earlier asked to remove this case from tests. 
Also, are_required_outputs shouldn't have an all-False value where the op returns no output tensors